src/components/Tabs.js

```diff
@@ -80,9 +80,13 @@ class Tabs extends Component {
     }
 
     render() {
-        const handleSelect = this.props.handleSelect || this.handleSelect;
-        const selectedTab = this.props.selectedTab ||
-            this.state.selectedTab ||
+        const {
+            handleSelect: propHandleSelect,
+            selectedTab: propSelectedTab,
+            ...tabsProps
```

Comment:
`tabsProps` can now still contain `name`, `activeLinkStyle` and `visibleTabStyle` - these need to be removed as well. After that it can be merged :+1: